packages/client/modules/userDashboard/components/OrgTeams/OrgTeamsRow.tsx

```diff
@@ -24,33 +27,70 @@ const OrgTeamsRow = (props: Props) => {
           isOrgAdmin
           isSelf
           email
+          preferredName
         }
       }
     `,
     teamRef
   )
   const {id: teamId, teamMembers, name} = team
   const teamMembersCount = teamMembers.length
+  const viewerTeamMember = teamMembers.find((m) => m.isSelf)
+  if (!viewerTeamMember && !isOrgAdmin) return null
```

Comment:
-1 billing leads should also see all teams, now we get this situation:
<img width="970" alt="image" src="https://github.com/user-attachments/assets/0f3d3bfc-edfd-4681-8a5a-ef0deb3f6676">